CONFIGURATION.md

```diff
@@ -13,7 +13,7 @@ The following environment variables can be set:
 | MONGO_URI             | MongoDB URI                 |                        |
 | MONGO_MIN_POOL_SIZE   | Min. connections per worker | 0                      |
 | MONGO_MAX_POOL_SIZE   | Max. connections per worker | 3                      |
-| MONGO_REPLICA_ENABLED | Read Replica (RR) Toggle    | 'false'                |
+| MONGO_REPLICA_ENABLED | Read Replica (RR) Toggle    | false                  |
 | TOR_HOST              | Tor Hostname                |                        |
 | TOR_CONTROL_PORT      | Tor control Port            | 9051                   |
 | TOR_PASSWORD          | Tor control password        | nostr_ts_relay         |
```

Comment:
确实 MONGO_MAX_POOL_SIZE 在现有情况下能够设置为负数，有什么好的修改建议吗

Reply:
> with the code review
> 
> 1. The change that was made to the MONGO_REPLICA_ENABLED variable is correct. The value of the variable should be set to false instead of 'false'.
> 2. It is good to use environment variables to set the MongoDB connection parameters. This makes it easier to change the values in the future if needed.
> 3. It is also important to add proper validation for all of the environment variables. For example, if the MONGO_MAX_POOL_SIZE variable is set to a negative number, this should be caught and an error should be thrown.
> 4. It would also be good to add comments for each of the environment variables to explain what each one does. This will help anyone who needs to modify the code in the future.

确实 MONGO_MAX_POOL_SIZE 在现有情况下能够设置为负数，有什么好的修改建议吗